### PR TITLE
Start up a dev server with gulp-nodemon

### PR DESCRIPTION
**Pull request by lmorchard**
Another small tweak, one less command to run. Gulp can now start up a nodemon process to run the app server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,7 @@ Basically a clone of [eb-node-express-signup](https://github.com/awslabs/eb-node
 
 ```
 npm install
-gulp // builds and starts watching files for rebuild
-npm start // better yet: nodemon server.js
+gulp // watches files for rebuild, starts an app server in dev mode
```
Comment (6a68) [inline README.md]: Should we keep `npm start` instead of keeping `gulp`?

**Comment by 6a68** [comment]
@lmorchard this is awesome!

File: README.md (hunk 1 above)
Comment (lmorchard) [inline README.md]: Well, keeping both is probably useful: `npm start` for just firing up the server can be handy for prod envs, `gulp` for the works on dev envs

File: gulpfile.js [hunk 2]
```diff
@@ -139,8 +140,16 @@ gulp.task('build', function (done) {
   );
 });
 
+gulp.task('server', ['build'], function () {
+  nodemon({
+    script: 'server.js',
+    ext: 'html js',
+    env: { 'NODE_ENV': 'development' }
+  });
```
Comment (pdehaan) [inline gulpfile.js]: I'm not a **nodemon** expert, but not sure if we want to recycle the existing [/nodemon.json](https://github.com/mozilla/idea-town/blob/master/nodemon.json) config file here.

Not that it matters much for the splash, but I'm not sure how much of this scaffolding we'll want to copy over for v2.

File: README.md (hunk 1 above)
Comment (6a68) [inline README.md]: Yeah, I slightly regretted picking that nit, but not quite enough to delete the comment ^_^

File: gulpfile.js (hunk 2 above)
Comment (6a68) [inline gulpfile.js]: > Not that it matters much for the splash, but I'm not sure how much of this scaffolding we'll want to copy over for v2.

Scorched earth, bro. Python.

**Comment by 6a68** [comment]
:tada:

File: gulpfile.js (hunk 2 above)
Comment (lmorchard) [inline gulpfile.js]: Oh, derp, I should have re-used the nodemon.json. Worth thinking about in future stuff I guess :)

**Pull request merged**